pkg/encoding/simple8b/encoding.go

```diff
@@ -541,7 +545,8 @@ func pack120(src []uint64) uint64 {
 
 // pack60 packs 60 values from in using 1 bit each
 func pack60(src []uint64) uint64 {
-	return 2<<60 |
+	return src[59]<<59 | // eliminate multiple bounds checks
```

Comment:
I would recommend leaving the order and adding this line to eliminate the bounds check, which follows precedent established by the Go team in [`encoding/binary` package](https://github.com/golang/go/blob/ad7ce3911f4947b4cccb435b6374e3808f0062db/src/encoding/binary/binary.go#L53).

```
_ = src[59] // bounds check hint to compiler; see golang.org/issue/14808
```